src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -278,22 +293,23 @@ private static async Task AddSubmissionDependentProjectsAsync(Solution solution,
         private static bool IsInternalsVisibleToAttribute(AttributeData attr)
         {
             var attrType = attr.AttributeClass;
-            if (attrType == null)
-            {
-                return false;
-            }
-
-            var attributeName = attr.AttributeClass.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat.WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted));
-            return attributeName == "System.Runtime.CompilerServices.InternalsVisibleToAttribute";
+            return attrType?.Name == nameof(InternalsVisibleToAttribute) &&
+                   attrType.ContainingNamespace?.Name == nameof(System.Runtime.CompilerServices) &&
+                   attrType.ContainingNamespace.ContainingNamespace?.Name == nameof(System.Runtime) &&
+                   attrType.ContainingNamespace.ContainingNamespace.ContainingNamespace?.Name == nameof(System) &&
+                   attrType.ContainingNamespace.ContainingNamespace.ContainingNamespace.ContainingNamespace?.IsGlobalNamespace == true;
```

Comment:
💭 It would be nice to have a helper `IsSameTypeName(Type)` that performs this check:

```csharp
return attrType.IsSameTypeName(typeof(InternalsVisibleToAttribute));
```

Reply:
idealy, this would just grab the "GetBestTypeByMetadataName" and hten just compare that type to the attrType.  But i was worried that might subtle change semantics.